crates/lox-space/examples/iss.rs

```diff
@@ -6,12 +6,13 @@
  * file, you can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
+use lox_core::time::utc::UTC;
 use lox_space::prelude::*;
 
 fn main() {
     let date = Date::new(2016, 5, 30).unwrap();
-    let time = Time::new(12, 0, 0).unwrap();
-    let epoch = Epoch::from_date_and_time(TimeScale::TDB, date, time);
+    let time = UTC::new(12, 0, 0).unwrap();
+    let epoch = ContinuousTime::from_date_and_utc_timestamp(ContinuousTimeScale::TDB, date, time);
```

Comment:
From an ergonomics perspective I am a bit dissatisfied with the `Continuous*` prefix. I believe that this gives the shortcomings and idiosyncrasies of UTC undue influence over the design.
In other words, `ContinuousTime` should be the default and thus probably be just `Time` (the current name is also a bit redundant since it is coming from the `continuous` module) and I am asking myself whether we really need the `TimeScale` enum.
UTC should be an I/O format only which is already very well reflected in this design except for the above. Let's treat it like the red-haired step child that it is 😬